examples/ensemble/plot_gradient_boosting_early_stopping.py

```diff
@@ -45,10 +45,12 @@
 from sklearn import datasets
 from sklearn.model_selection import train_test_split
 
-data_list = [datasets.load_iris(), datasets.load_digits()]
-data_list = [(d.data, d.target) for d in data_list]
-data_list += [datasets.make_hastie_10_2()]
-names = ["Iris Data", "Digits Data", "Hastie Data"]
+data_list = []
+
+data_list += [datasets.load_iris(return_X_y=True)]
+data_list += [datasets.make_classification(n_samples=800)]
+data_list += [datasets.make_hastie_10_2(n_samples=2000)]
```

Comment:
We can set the `random_state` to have deterministic runs:

```suggestion
data_list = [
    datasets.load_iris(return_X_y=True),
    datasets.make_classification(n_samples=800, random_state=0),
    datasets.make_hastie_10_2(n_samples=2000, random_state=0),
]
```

Also, we can create the list directly.

Reply:
Seems reasonable. Thanks for the suggestion!